README.md

```diff
@@ -159,11 +159,16 @@ To list all available targets, run
 
     fusesoc core show swervolf
 
+To build and run on Riviera-Pro simulator
+
+    fusesoc run --target=sim --tool=rivierapro swervolf --compilation_mode=common
+
```

Comment:
As I mentioned in the other commit, set compilation mode directly in the core file instead of having to set it on the command-line